src/utils/__tests__/__snapshots__/publicAPI.test.js.snap

```diff
@@ -61,36 +23,6 @@ Map {
       "kind": "primary",
       "loading": false,
     },
-    "propTypes": Object {
```

Comment:
I'm seeing most (if not all) propTypes removed from the api snapshot.  Is this intentional?  I thought the purpose of this file was to compare the APIs from build to build, release to release, etc.

Reply:
Yes this is intended, but we shouldn't be stripping them completely. They should be wrapped in a conditional, [see my review](https://github.com/IBM/carbon-addons-iot-react/pull/1036#discussion_r398787376) so consuming applications still get proptype warnings.

There is [a bug with Terser](https://github.com/terser/terser/issues/293), with [a decent workaround synopsis here](https://github.com/alex996/react-css-spinners/issues/1#issuecomment-513634336), where tree shaking breaks when 2 or more static properties are assigned to a function. Our components typically have `defaultProps` and `propTypes`. [Carbon ran into the same issue recently](https://github.com/carbon-design-system/carbon/issues/5442) causing them to revert an update to their build.